hack/jenkins/upload-to-gcs.sh

```diff
@@ -46,8 +46,22 @@ if [[ ! ${JENKINS_UPLOAD_TO_GCS:-y} =~ ^[yY]$ ]]; then
   exit 0
 fi
 
+GCS_SUBDIR=""
+readonly remote_git_repo=$(git config --get remote.remote.url | sed 's:.*github.com/::' || true)
+if [[ -n "${remote_git_repo}" ]]; then
+  case "${remote_git_repo}" in
+    kubernetes/kubernetes) GCS_SUBDIR="" ;;
+    kubernetes/*) GCS_SUBDIR="${remote_git_repo#kubernetes/}/" ;;
```

Comment:
do we want this case? or should we just treat everything that's not `kubernetes/kubernetes` with its full path?